shared/types/src/dataset.ts

```diff
@@ -1086,27 +1071,51 @@ type AssayAvailability = {
 
 //Shared with genome.ts
 export type Cohort = {
-	hideGroupsTab?: boolean
-	tabs?: { [index: string]: string } //Customizes tab top label
-	/** customize the default chart to open on mass ui when there's no charts. if missing it opens dictionary ui */
-	defaultChartType?: string
 	allowedChartTypes?: string[]
-	hiddenChartTypes?: string[]
-	renamedChartTypes?: { singleCellPlot?: string; sampleScatter?: string }
-	mutationset?: MutationSet[]
-	db: FileObj
-	termdb: Termdb
-	scatterplots?: Scatterplots
-	matrixplots?: MatrixPlots
-	/** optional title of this ds, if missing use ds.label. shown on mass nav header. use blank string to not to show a label*/
-	title?: Title
 	cumburden?: {
 		files: {
 			fit: string
 			surv: string
 			sample: string
 		}
 	}
+	db: FileObj
+	/** customize the default chart to open on mass ui when there's no charts. if missing it opens dictionary ui */
+	defaultChartType?: string
```

Comment:
Maybe we could also move the defaultChartType and the hiddenChartTypes to customizations, as they customize the dataset behaviour?

Reply:
It's a good idea. It might best to arrange that under a `.massCharts`, or something similar, obj so it's crystal clear. That's could be a future PR.